### PR TITLE
Update init modal design

### PR DESCRIPTION
**Pull request by thomasguillot**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR redesigns the `Modal` on init and removes the big preview area to focus on the layouts instead.

my-layouts

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Remove all the Newsletter settings so you can also see the 1st step
3. Create a new newsletter 

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: src/components/init-modal/screens/api-keys/index.js [hunk 1]
```diff
@@ -109,19 +109,18 @@ export default ( { onSetupStatus } ) => {
 								disabled: true,
 								label: __( 'Select service provider', 'newspack-newsletters' ),
 							},
-							{ value: 'manual', label: __( 'Manual', 'newspack-newsletters' ) },
-							{ value: 'mailchimp', label: __( 'Mailchimp', 'newspack-newsletters' ) },
+							{ value: 'mailchimp', label: 'Mailchimp' },
```
Comment (dkoo) [inline src/components/init-modal/screens/api-keys/index.js:112]: Why remove the translation on these strings?

File: src/components/init-modal/screens/api-keys/index.js [hunk 2]
```diff
@@ -109,19 +109,18 @@ export default ( { onSetupStatus } ) => {
...
 							{
 								value: 'constant_contact',
-								label: __( 'Constant Contact', 'newspack-newsletters' ),
+								label: 'Constant Contact',
```
Comment (dkoo) [inline src/components/init-modal/screens/api-keys/index.js:115]: Same question: why no translation?

File: src/components/init-modal/screens/api-keys/index.js [hunk 3]
```diff
@@ -109,19 +109,18 @@ export default ( { onSetupStatus } ) => {
...
 							},
 							{
 								value: 'campaign_monitor',
-								label: __( 'Campaign Monitor', 'newspack-newsletters' ),
+								label: 'Campaign Monitor',
```
Comment (dkoo) [inline src/components/init-modal/screens/api-keys/index.js:119]: Same question: why no translation?

File: src/newsletter-editor/editor/style.scss [hunk 4]
```diff
@@ -2,7 +2,9 @@
 	.editor-post-title {
 		&.newspack-newsletters-post-title-hidden {
 			height: 0;
+			margin: -4rem 0 calc( -1 * var( --wp--style--block-gap ) );
```
Comment (dkoo) [inline src/newsletter-editor/editor/style.scss:5]: This negative margin causes the content to get cut-off at the top if your layout doesn't have the logo image as the first block:

Screen Shot 2022-02-01 at 4 41 10 PM

File: src/components/init-modal/style.scss [hunk 5]
```diff
@@ -294,53 +359,6 @@
 	}
 }
 
-// Buttons group
-.newspack-newsletters-buttons-group {
-	display: flex;
-	justify-content: space-around;
-	&--spaced {
```
Comment (dkoo) [inline src/components/init-modal/style.scss:301]: Getting rid of these styles causes the buttons in the Layout sidebar to look smushed together:

Screen Shot 2022-02-01 at 4 44 45 PM

**Review by dkoo — CHANGES_REQUESTED** [review]
I like the new modal layout—just ran into a few minor issues below.

File: src/components/init-modal/screens/api-keys/index.js (hunk 1 above)
Comment (thomasguillot) [reply src/components/init-modal/screens/api-keys/index.js:112]: Mailchimp is a brand so it shouldn't be translated.

File: src/components/init-modal/screens/api-keys/index.js (hunk 2 above)
Comment (thomasguillot) [reply src/components/init-modal/screens/api-keys/index.js:115]: Constant Contact is a brand so it shouldn't be translated.

File: src/components/init-modal/screens/api-keys/index.js (hunk 3 above)
Comment (thomasguillot) [reply src/components/init-modal/screens/api-keys/index.js:119]: Campaign Monitor is a brand so it shouldn't be translated.

File: src/newsletter-editor/editor/style.scss (hunk 4 above)
Comment (thomasguillot) [reply src/newsletter-editor/editor/style.scss:6]: Fixed: https://github.com/Automattic/newspack-newsletters/pull/710/commits/47f2f4e74cab7ae11fa951c1ab6b719eccb2ac8b

File: src/components/init-modal/style.scss (hunk 5 above)
Comment (thomasguillot) [reply src/components/init-modal/style.scss:301]: Fixed: https://github.com/Automattic/newspack-newsletters/pull/710/commits/2a5476a70161c0e8c74abccd241321f12cfee3b1

**Review by dkoo — APPROVED** [review]
Looks better, thanks for the updates 👍 

Screen Shot 2022-02-02 at 10 51 26 AM

**Pull request merged**